vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/StatusList.kt

```diff
@@ -0,0 +1,57 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList
+
+import at.asitplus.wallet.lib.DefaultZlibService
+import at.asitplus.wallet.lib.ZlibService
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.TokenStatusBitSize
+import kotlinx.serialization.Serializable
+
+@Serializable(with = StatusListSerializer::class)
+data class StatusList(
```

Comment:
Please add code documentation for this class and `Status` and `StatusListInfo` and `StatusListView` on how they're connected with each other and what represents what